tsdb/chunks/head_chunks.go

```diff
@@ -273,6 +273,25 @@ func NewChunkDiskMapper(reg prometheus.Registerer, dir string, pool chunkenc.Poo
 	return m, m.openMMapFiles()
 }
 
+// Chunk encodings for out-of-order chunks.
+// These encodings must be only used by the Head block for its internal bookkeeping.
+const (
+	OutOfOrderMask        = 0b10000000
+	RestoreOutOfOrderMask = 0b01111111
+)
+
+func (cdm *ChunkDiskMapper) OutOfOrderEncoding(sourceEncoding chunkenc.Encoding) chunkenc.Encoding {
```

Comment:
I would rename this to `ApplyOutOfOrderEncoding` or something like that since its actually applying a mask over the giving encoding.

Reply:
Ok, will rename it as suggested.

Reply:
Ok, will do